packages/react-composites/src/composites/common/CallingDialpad.tsx

```diff
@@ -43,8 +47,22 @@ export const CallingDialpad = (props: CallingDialpadProps): JSX.Element => {
   };
 
   const onClickCall = (): void => {
-    //place holder for adding calling functionality
-    console.log(textFieldInput);
+    if (onAddParticipant) {
+      let phoneNumber;
+      /**
+       * Format the phone number in dialpad textfield to make sure the phone number is in E.164 format.
+       * PSTN is only for North America and no area code in NA start with 1
+       * Check if the phone number input start with 1, if yes, the input phone number already contains country code, add + on top
+       * If not, add +1 country code
+       */
+      if (textFieldInput.replace(/\D/g, '')[0] !== '1') {
+        phoneNumber = { phoneNumber: '+1' + textFieldInput.replace(/\D/g, '').replaceAll(' ', '') };
```

Comment:
Did we want to go forward with this in the composite? I wonder if we could expose a prop to the composite or something to expose a optional country code so that we don't lock them into NA only? @anjulgarg do you have any thoughts on this?

Reply:
Currently PSTN is a NA only feature

Reply:
Thanks for noticing. Are we not asking customers to provide the phone number with a country code?